config/database.yml

```diff
@@ -61,6 +61,8 @@ development:
 test:
   <<: *default
   database: kick_test
+  username: user-wbooks-test
+  password: user-wbooks-test
```

Comment:
Wait, didn't you add this in a previous PR?

And the database name is different. In the previous PR you changed it to `wbooks`